src/server/onboarding-router/router.ts

```diff
@@ -0,0 +1,10 @@
+import { createTRPCRouter, protectedProcedure } from "../api/trpc";
+import { ZOnboardMutationSchema } from "./schema";
```

Comment:
`./schema` has `TrpcOnboardMutationSchema` and `TOnboardMutationSchema` exported, but you are importing `ZOnboardMutationSchema`, i am confused. Perhaps a typo when trying to find and replace? 🤔

Reply:
Also, curious if it makes more sense to keep these files under `trpc` directory instead of server? 🤔

Reply:
> Also, curious if it makes more sense to keep these files under `trpc` directory instead of server? 🤔

it should make sense since every trpc related code can be colocated inside a specific folder